notifications_utils/recipients.py

```diff
@@ -290,13 +345,12 @@ def is_optional_address_column(self, key):
 
     @property
     def has_recipient_columns(self):
-        return (
+        return set([list(self.column_headers_as_column_keys)[0]]).issubset(
```

Comment:
What does this do? Should we have a comment here maybe?

Reply:
done